partcad/tests/unit/test_git_clone_retries.py

```
@@ -0,0 +1,128 @@
+        raise git_error
+    side_effect.counter = 0
+
+    with patch("git.Repo.clone_from", side_effect=side_effect) as mock_clone_from, \
+         patch.object(pc.user_config, "git_retry_config", test_git_retry_config), \
+         tempfile.TemporaryDirectory() as temp_dir, \
+         patch.object(pc.user_config, "internal_state_dir", temp_dir), \
+         pytest.raises(RuntimeError):
+
+        ctx = pc.Context()
+        factory = pc.ProjectFactoryGit(ctx, None, test_config_import_git)
+        factory._clone_or_update_repo(repo_url)
```

Ruff (0.8.2)

`pytest.raises()` block should contain a single simple statement

(PT012)